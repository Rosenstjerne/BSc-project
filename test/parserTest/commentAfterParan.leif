var int a,b;
var bool c;
a = 1;
b = 2;
c = true;

print(1+2)#* test *#;