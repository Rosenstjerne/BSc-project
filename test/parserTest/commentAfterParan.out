function null main() {
	var int a, b;
	
	a = 1;
	b = 2;
	print((a + b));
	
}
